codegen/scope.go

```diff
@@ -67,22 +67,22 @@ done:
 
 // GoTypeDef returns the Go code that defines a Go type which matches the data
 // structure definition (the part that comes after `type foo`).
-func (s *NameScope) GoTypeDef(att *design.AttributeExpr, useDefault bool) string {
+func (s *NameScope) GoTypeDef(att *design.AttributeExpr, useDefault bool, pointer bool) string {
```

Comment:
Would it make things easier if there was two methods `GoTypeDef` and `GoTypeDefPointer` instead of an argument? Maybe it would have helped but now that the change is made it doesn't help anymore.

Reply:
Well.. I am ok having it either way. Right now, adding the `pointer` parameter is not complicating the existing code too much. I don't see us adding much more to this code.